### PR TITLE
webgl-utils README. break out getParameter polyfill.

### PR DESCRIPTION
**Pull request by ibgreen**
@1chandu - Not yet ready for formal review, but please take a look. I may need some help wrapping this up.

* Extract the `gl.getParameter` polyfill - this should significantly simplify `webgl-utils/polyfill-context.js` as well as `webgl/context-limits.js`

* Updates the `README.md` for` webgl-utils` directory (as these functions are not described in the "official" docs)

### CODE REVIEW

File: src/webgl-utils/polyfill-get-parameter.js [hunk 1]
```diff
@@ -9,23 +9,25 @@ const EXT_disjoint_timer_query_webgl2 = 'EXT_disjoint_timer_query_webgl2';
 const EXT_texture_filter_anisotropic = 'EXT_texture_filter_anisotropic';
 const WEBGL_debug_renderer_info = 'WEBGL_debug_renderer_info';
 
-const GL_FRAGMENT_SHADER_DERIVATIVE_HINT = 0x8B8B;
-const GL_DONT_CARE = 0x1100;
-const GL_GPU_DISJOINT_EXT = 0x8FBB;
-const GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT = 0x84FF;
-const GL_UNMASKED_VENDOR_WEBGL = 0x9245; // vendor string of the graphics driver.
-const GL_UNMASKED_RENDERER_WEBGL = 0x9246; // renderer string of the graphics driver.
+export const GL_FRAGMENT_SHADER_DERIVATIVE_HINT = 0x8B8B;
```
Comment (ibgreen) [inline src/webgl-utils/polyfill-get-parameter.js]: These should not be exported. We added these local constants only to enable minification. 

If you export them, you defeat that. 

In other places you typically use GL.<constant>

File: test/webgl-utils/polyfill-get-parameter.spec.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import {getParameterPolyfill} from '../../src/webgl-utils/polyfill-get-parameter';
+import GL from '../../src/webgl-utils/constants';
+import test from 'tape-catch';
+
+import {fixture} from '../setup';
+
+const WEBGL_PARAMETER_KYES = [
```
Comment (ibgreen) [inline test/webgl-utils/polyfill-get-parameter.spec.js]: KYES => KEYS (typo)

File: test/webgl-utils/polyfill-get-parameter.spec.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+];
+
+test('getParameterPolyfill#import', t => {
+  t.ok(typeof getParameterPolyfill === 'function', 'getParameterPolyfill imported OK');
+  t.end();
+});
+
+test('getParameterPolyfill#WebGL1', t => {
+  const {gl} = fixture;
+  const originalGetParameterFunc = gl.getParameter.bind(gl);
```
Comment (ibgreen) [inline test/webgl-utils/polyfill-get-parameter.spec.js]: Yes looks right!

**Comment by 1chandu** [comment]
Authored and reviewed by Ib and myself.

**Pull request merged**